packages/amplify-e2e-tests/src/rds-v2-tests-common/rds-auth-userpool-fields.ts

```diff
@@ -0,0 +1,1925 @@
+
+      /*
+      // non-owner cannot read a record with dynamic owner list protected field in the selection set
+      const ownerReadFieldSet = `
+        customId
+        authors
+        ownersContent
+      `;
+      const getResult2 = await user2TodoHelper.get({ customId: todo['customId'] }, ownerReadFieldSet, false, 'all', 'customId');
+      checkOperationResult(
+        getResult2, 
+        { customId: todo['customId'], authors: null, ownersContent: null }, 
+        `get${modelName}`, 
+        false, 
+        expectedFieldErrors(['authors', 'ownersContent'], modelName)
+      );
+
+      const listTodosResult2 = await user2TodoHelper.list({}, ownerReadFieldSet, `list${modelName}`, false, 'all');
+      checkListItemExistence(listTodosResult2, `list${modelName}`, todo['customId'], true, 'customId');
+      checkListResponseErrors(listTodosResult2, expectedFieldErrors(['authors', 'ownersContent'], modelName, false));
+      */
+
+      // non-owner cannot listen to updates on owner protected fields
+      const todoRandom = {
+        ...todo,
+        customId: Date.now().toString(),
+      };
+      const todoRandomUpdated = {
+        customId: todoRandom.customId,
+        authors: [userName1],
+        privateContent: 'Private Content updated',
+        ownersContent: 'Owners Content updated',
+      };
+      const subscriberClient = getConfiguredAppsyncClientCognitoAuth(graphQlEndpoint, region, userMap[userName2]);
+      const subTodoHelper = createModelOperationHelpers(subscriberClient, schema)[modelName];
+
+      /*
+      const onCreateSubscriptionResult = await subTodoHelper.subscribe(
+        'onCreate', [
+          async () => {
+            await user1TodoHelper.create(createResultSetName, todoRandom, completeOwnerResultSet);
+          },
+        ],
+        {},
+        completeOwnerResultSet,
+        false,
+      );
+      expect(onCreateSubscriptionResult).toHaveLength(1);
+      expect(onCreateSubscriptionResult[0].data[`onCreate${modelName}`].customId).toEqual(todoRandom.customId);
+      expectNullFields(onCreateSubscriptionResult[0].data[`onCreate${modelName}`], ['authors', 'privateContent', 'ownersContent']);
+
+      const onUpdateSubscriptionResult = await subTodoHelper.subscribe(
+        'onUpdate', 
+        [
+          async () => {
+            await user1TodoHelper.update(updateResultSetName, todoRandomUpdated, completeOwnerResultSet);
+          },
+        ],
+        {},
+        completeOwnerResultSet,
+        false,
+      );
+      expect(onUpdateSubscriptionResult).toHaveLength(1);
+      expect(onUpdateSubscriptionResult[0].data[`onUpdate${modelName}`].customId).toEqual(todoRandom.customId);
+      expectNullFields(onUpdateSubscriptionResult[0].data[`onUpdate${modelName}`], ['authors', 'privateContent', 'ownersContent']);
+      */
+    });
+
+    test.skip('non-admin users can perform CRUD and subscription operations with only private fields in selection set', async () => {
```

Comment:
As above, add an easily searchable todo, or link to this code in our backlog ticket